src/reconciler.tsx

```diff
@@ -114,16 +114,21 @@ export const extend = (objects: object): void => void (catalogue = { ...catalogu
 export function applyProps(instance: any, newProps: any, oldProps: any = {}, accumulative: boolean = false) {
   // Filter equals, events and reserved props
   const container = instance.__container
-  const sameProps = Object.keys(newProps).filter((key) => is.equ(newProps[key], oldProps[key]))
-  const handlers = Object.keys(newProps).filter((key) => {
```

Comment:
Here we are performing a redundant operation of `Object.keys(newProps)` in 2 areas. We're also filtering through all of the keys twice. It is faster to use `reduce` and mutate the accumulator so that we are performing both filters in the same pass. This is typically known as a transducer, IIRC.